src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/felles/pdf/JournalføringService.kt

```diff
@@ -103,10 +103,16 @@ class JournalføringService(
 
     private fun lagMottager(behandling: Behandling, mottager: Brevmottager, brevmetadata: Brevmetadata): AvsenderMottaker {
         val adresseinfo: Adresseinfo = brevmetadata.mottageradresse
+        val mottagerIdent = adresseinfo.ident.takeIf { it.isNotEmpty() }
         return when (mottager) {
-            Brevmottager.BRUKER -> AvsenderMottaker(
-                id = adresseinfo.ident,
-                idType = BrukerIdType.FNR,
+            Brevmottager.BRUKER, Brevmottager.MANUELL -> AvsenderMottaker(
+                id = mottagerIdent,
+                idType = when (mottagerIdent?.length) {
+                    0, null -> null
+                    9 -> BrukerIdType.ORGNR
+                    11 -> BrukerIdType.FNR
+                    else -> throw IllegalArgumentException("Ugyldig idType")
+                },
```

Comment:
Lurer på om man kan trekke denne ut til en egen metode, slik at man ikke har en when i en when.

```suggestion
                idType = hentIdType(mottagerIdent?.length
                 ,
```